fallback_builtins.h

```diff
@@ -71,6 +71,24 @@ static inline __m512i _mm512_zextsi128_si512(__m128i a) {
 
 #endif // __AVX2__
 
+/* Missing zero-extension AVX and AVX512 intrinsics.
+ * Fixed in version 15.7
+ * https://developercommunity.visualstudio.com/t/missing-zero-extension-avx-and-avx512-intrinsics/175737
+ */
+#if defined(_MSC_VER) && _MSC_VER < 1914
+#ifdef X86_AVX2
+static inline __m256i _mm256_zextsi128_si256(__m128i a) {
+    return _mm256_inserti128_si256(_mm256_setzero_si256(), a, 0);
+}
+#endif // X86_AVX2
```

Comment:
This gives a warning on line 81 complaining that `/arch:AVX` isn't used. Maybe test for `__AVX2__` to see the code doesn't get included without proper compiler option.

Reply:
zlib-ng buildsystem does not set ``/arch:AVX*`` on MSVC.
I'll fix it in a separate commit.

Reply:
Fixed.